modMcf/src/org/aion/mcf/core/ImportResult.java

```diff
@@ -46,4 +46,8 @@ public boolean isNotBest() {
     public boolean isStored() {
         return equals(EXIST) || isSuccessful();
     }
+
+    public boolean isValid() {
+        return !equals(INVALID_BLOCK) && !equals(CONSENSUS_BREAK);
```

Comment:
I would write this as

 `!(equals(INVALID_BLOCK) || equals(CONSENSUS_BREAK)`

 This is my only suggested change :P

Reply:
do you mean that `CONSENSUS_BREAK` should be considered valid?

Reply:
this is just the logic gate combination. very small performance enhancement :)

Reply:
Yeah, what Jay said...I would always prefer to write !(X or Y) instead of (!X and !Y).

It was only a half-serious suggestion x)

Reply:
I'm not seeing ``CONSENSUS_BREAK`` used anywhere outside of tests

Reply:
ok, will update it

Reply:
@qoire so, you would prefer that we don't include it here?